src/Hooks.ts

```diff
@@ -0,0 +1,5 @@
+import { useDispatch,useSelector } from "react-redux";
+import { AppDispatch,RootState } from './store';
+
+export const useAppDispatch = useDispatch.withTypes<AppDispatch>();
+export const useAppSelector = useSelector.withTypes<RootState>();
```

Comment:
Create a hooks folder then and index.ts file inside that will export hooks in that folder. In the hooks folder you can then create a file for this then export everything from the index file